WcaOnRails/app/views/static_pages/score_tools.html.erb

```diff
@@ -1,9 +1,17 @@
-<% provide(:title, 'WCA Score Taking tools') %>
+<% provide(:title, t('score_tools.title')) %>
+
+<%
+tools_variables = {
+  workbook_link: link_to("WCA Workbook Assistant", wca_workbook_assistant_path)
+}
+%>
```

Comment:
I was going to do that as well, but followed Philippe's style at the faq_variables.